### PR TITLE
src/layouts: add image background to the LoginRegister layout

### PR DESCRIPTION
**Pull request by maceksimon**
Add background image with mask to login and register layouts (see design image)

* Update `LoginRegisterLayout` - add image.
* Update individual pages (remove background that covered up the layout elements.
* Add image in public folder with an URL passed via global config for easier replacement (if needed).
* Add image test to the E2E tests.

### CODE REVIEW

File: src/layouts/LoginRegisterLayout.vue [hunk 1]
```diff
@@ -37,12 +38,41 @@ if (window.Cypress) {
...
 });
 </script>
 
 <template>
-  <q-layout view="hHh lpR fFf">
-    <q-page-container>
+  <q-layout class="bg-primary" view="hHh lpR fFf">
+    <!-- Background image -->
+    <div
+      v-if="$q.screen.gt.md"
```
Comment (tmszi) [inline src/layouts/LoginRegisterLayout.vue:57]: `$q.screen.gt.md` return `true` if screen width is >= 1440 px, is this condition correct?

My tested configuration is monitor with 1366x768 px resolution, and background image is not rendered due screen width condition.

Comment (tmszi) [reply src/layouts/LoginRegisterLayout.vue:57]: And e2e tests doesn't catch this case.

Comment (maceksimon) [reply src/layouts/LoginRegisterLayout.vue]: Yes, this was the setup. But, as you write, it seems more appropriate to start the "desktop" layout on 
1024px breakpoint and above.
I updated the condition to `v-if="$q.screen.gt.sm"` and edited the column layout for register and login page accordingly.
Added an E2E image test for different resolutions.

Comment (tmszi) [reply src/layouts/LoginRegisterLayout.vue]: Looks good to me.

**Pull request merged**